test/services/VatPartialBuilderSpec.scala

```diff
@@ -40,19 +40,19 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 class VatPartialBuilderSpec
-    extends ViewSpecBase
+  extends ViewSpecBase
```

Comment:
can we check the format on this file
```suggestion
    extends ViewSpecBase
```